src/infra/AnalyticsReporter.ts

```diff
@@ -8,8 +8,12 @@ import { Visitor } from '../models/Visitor';
 
 const defaultDomain = 'https://answers.yext-pixel.com';
 
+/**
+ * @public
+ */
 export class AnalyticsReporter implements AnalyticsService {
   private _visitor: Visitor | undefined;
+
   constructor(private config: AnalyticsConfig, private httpRequesterService: HttpRequesterService) {
```

Comment:
Should this constructor be marked as internal?

Reply:
Going to make the whole component internal per Tom's comment